modules/apps/portal-vulcan/portal-vulcan-api/src/main/java/com/liferay/portal/vulcan/util/UriInfoUtil.java

```diff
@@ -46,13 +49,15 @@ public static String getBasePath(UriInfo uriInfo) {
 	}
 
 	public static UriBuilder getBaseUriBuilder(UriInfo uriInfo) {
```

Comment:
I see `getBaseUriBuilder` and `getAbsolutePath` are almost identical. Would it make sense to reuse the logic with a private method that receives the `uriBuilder`? This way if we have to fix something else again in the future we won't have to do it twice. What do you think?

Reply:
You are right. Updated!